packages/wdio-cli/package.json

```diff
@@ -53,9 +53,6 @@
     "webdriverio": "^5.0.0-beta.12",
     "yargs": "^11.1.0"
   },
-  "peerDependencies": {
```

Comment:
webdriverio is defined as an explicit dependency and a peer dependency. We only need the explicit dependency.